File: are-we-alternate/__tests__/areWeAlternate.test.js
const { isAlt } = require('../areWeAlternate');

describe('Given a string, return true if the vowels & consonants are in opposite order', () => {
    test('Given "amazon", return true', () => {
        const string = 'amazon';
        expect(isAlt(string)).toBe(true);
    });
    test('Given "apple", return false', () => {
        const string = 'apple';
        expect(isAlt(string)).toBe(false);
    });
    test('Given "banana", return true', () => {
        const string = 'banana';
        expect(isAlt(string)).toBe(true);
    });
    test('Given "fcfgf", return false', () => {
        const string = 'fcfgf';
        expect(isAlt(string)).toBe(false);
    });
    test('Given "hegeg", return true', () => {
        const string = 'hegeg';
        expect(isAlt(string)).toBe(true);
    });
    test('Given "frkvmgdrgjdnmkddvdaxcxwgceh", return false', () => {
        const string = 'frkvmgdrgjdnmkddvdaxcxwgceh';
        expect(isAlt(string)).toBe(false);
    });
    test('Given "atatatatatatatatatatatata", return true', () => {
        const string = 'atatatatatatatatatatatata';
        expect(isAlt(string)).toBe(true);
    });
});

//https://www.codewars.com/kata/59325dc15dbb44b2440000af/train/javascript

File: python-venv/are_we_alternate/__tests__/are_we_alternate_test.py
from kata_function.are_we_alternate import are_we_alternate

def test_1():
    string = 'amazon'
    assert are_we_alternate(string) == True

def test_2():
    string = 'apple'
    assert are_we_alternate(string) == False

def test_3():
    string = 'banana'
    assert are_we_alternate(string) == True

def test_4():
    string = 'fcfgf'
    assert are_we_alternate(string) == False

def test_5():
    string = 'hegeg'
    assert are_we_alternate(string) == True

def test_6():
    string = 'frkvmgdrgjdnmkddvdaxcxwgceh'
    assert are_we_alternate(string) == False

def test_7():
    string = 'atatatatatatatatatatatata'
    assert are_we_alternate(string) == True


#https://www.codewars.com/kata/59325dc15dbb44b2440000af/train/python